### PR TITLE
Videomaker: Add more patterns

### LINKED ISSUE #4859: Videomaker: Additional Block Patterns
In the meantime, after development started, we created a few more block patterns [in Figma here](https://www.figma.com/file/XnDSyzD9cBG5kbDaVZn9yG/Videomaker?node-id=183%3A563):

<img width="706" alt="Captura de ecrã 2021-10-18, às 16 28 40" src="https://user-images.githubusercontent.com/65220155/137774004-1ae81642-93e2-42d4-a05e-0d2f7f23e097.png">

I've built them in the editor. Here's the code for each of them:

**Film Credits**
```
{<!-- wp:spacer {"height":50} -->
<div style="height:50px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:paragraph {"align":"center","style":{"typography":{"lineHeight":"0.5"}},"fontSize":"medium"} -->
<p class="has-text-align-center has-medium-font-size" style="line-height:0.5">Based on the novel “Man Working at Dawn” by</p>
<!-- /wp:paragraph -->

<!-- wp:paragraph {"align":"center","style":{"typography":{"lineHeight":"0.5"}},"fontSize":"large"} -->
<p class="has-text-align-center has-large-font-size" style="line-height:0.5">LOIS HERBERT</p>
<!-- /wp:paragraph -->

<!-- wp:spacer {"height":20} -->
<div style="height:20px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:paragraph {"align":"center","style":{"typography":{"lineHeight":"0.5"}},"fontSize":"medium"} -->
<p class="has-text-align-center has-medium-font-size" style="line-height:0.5">Screenplay by</p>
<!-- /wp:paragraph -->

<!-- wp:paragraph {"align":"center","style":{"typography":{"lineHeight":"1.3"}},"fontSize":"large"} -->
<p class="has-text-align-center has-large-font-size" style="line-height:1.3">AUGUSTA SMITH<br>DAN ANDREWS<br>GARY HAN</p>
<!-- /wp:paragraph -->

<!-- wp:spacer {"height":20} -->
<div style="height:20px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:paragraph {"align":"center","style":{"typography":{"lineHeight":"0.5"}},"fontSize":"medium"} -->
<p class="has-text-align-center has-medium-font-size" style="line-height:0.5">Directed by</p>
<!-- /wp:paragraph -->

<!-- wp:paragraph {"align":"center","style":{"typography":{"lineHeight":"1.3"}},"fontSize":"large"} -->
<p class="has-text-align-center has-large-font-size" style="line-height:1.3">GARY HAN</p>
<!-- /wp:paragraph -->

<!-- wp:spacer {"height":50} -->
<div style="height:50px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->}
```

**Description**

```
{<!-- wp:spacer {"height":50} -->
<div style="height:50px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:paragraph {"align":"center","fontSize":"large"} -->
<p class="has-text-align-center has-large-font-size">Man Working at Dawn</p>
<!-- /wp:paragraph -->

<!-- wp:columns {"isStackedOnMobile":false} -->
<div class="wp-block-columns is-not-stacked-on-mobile"><!-- wp:column -->
<div class="wp-block-column"><!-- wp:paragraph {"align":"right","fontSize":"medium"} -->
<p class="has-text-align-right has-medium-font-size"><strong>Release Date<br>Rating</strong><br><br><strong>Year<br>Country<br>Slogan<br>Genre<br>Time<br>Budget<br>Age</strong></p>
<!-- /wp:paragraph --></div>
<!-- /wp:column -->

<!-- wp:column -->
<div class="wp-block-column"><!-- wp:paragraph {"fontSize":"medium"} -->
<p class="has-medium-font-size">25th March 2020<br>PG-13<br><br>2016<br>United States of America<br>“A Life of Labor”<br>Short Film, Documentary<br>35 min.<br>$ 450 000<br>12+</p>
<!-- /wp:paragraph --></div>
<!-- /wp:column --></div>
<!-- /wp:columns -->

<!-- wp:spacer {"height":50} -->
<div style="height:50px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->}
```


**Awards**

```
{<!-- wp:cover {"overlayColor":"background","minHeight":100,"minHeightUnit":"vh","align":"wide","style":{"spacing":{"padding":{"top":"50px","right":"50px","bottom":"50px","left":"50px"}}}} -->
<div class="wp-block-cover alignwide has-background-background-color has-background-dim" style="padding-top:50px;padding-right:50px;padding-bottom:50px;padding-left:50px;min-height:100vh"><div class="wp-block-cover__inner-container"><!-- wp:columns {"verticalAlignment":"center","style":{"spacing":{"blockGap":"70px"}}} -->
<div class="wp-block-columns are-vertically-aligned-center"><!-- wp:column {"verticalAlignment":"center","style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}}}} -->
<div class="wp-block-column is-vertically-aligned-center" style="padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:image {"id":50,"sizeSlug":"full","linkDestination":"none"} -->
<figure class="wp-block-image size-full"><img src="https://videomakerdemo.files.wordpress.com/2021/10/best_act.png" alt="" class="wp-image-50"/></figure>
<!-- /wp:image --></div>
<!-- /wp:column -->

<!-- wp:column {"verticalAlignment":"center","style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}}}} -->
<div class="wp-block-column is-vertically-aligned-center" style="padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:image {"id":51,"sizeSlug":"full","linkDestination":"none"} -->
<figure class="wp-block-image size-full"><img src="https://videomakerdemo.files.wordpress.com/2021/10/new_film.png" alt="" class="wp-image-51"/></figure>
<!-- /wp:image --></div>
<!-- /wp:column -->

<!-- wp:column {"verticalAlignment":"center","style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}}}} -->
<div class="wp-block-column is-vertically-aligned-center" style="padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:image {"id":52,"sizeSlug":"full","linkDestination":"none"} -->
<figure class="wp-block-image size-full"><img src="https://videomakerdemo.files.wordpress.com/2021/10/best_short.png" alt="" class="wp-image-52"/></figure>
<!-- /wp:image --></div>
<!-- /wp:column --></div>
<!-- /wp:columns --></div></div>
<!-- /wp:cover -->}
```

**Reviews**

```
{<!-- wp:spacer -->
<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:columns {"align":"wide"} -->
<div class="wp-block-columns alignwide"><!-- wp:column -->
<div class="wp-block-column"><!-- wp:paragraph {"fontSize":"medium"} -->
<p class="has-medium-font-size"><em>“Han’s newest short film is an absurdly and assuredly packed punch in the gut that’s short, shocking, and, incidentally, a romantic view on underdeveloped coutries.”</em></p>
<!-- /wp:paragraph -->

<!-- wp:paragraph {"fontSize":"small"} -->
<p class="has-small-font-size"><strong>— Latest Review</strong></p>
<!-- /wp:paragraph --></div>
<!-- /wp:column -->

<!-- wp:column -->
<div class="wp-block-column"><!-- wp:paragraph {"fontSize":"medium"} -->
<p class="has-medium-font-size"><em>“The film is beautifully shot and captures the picturesque African savannah, with the memorable image of a man working at dawn, creating a striking contrast between nature and humankind.”</em></p>
<!-- /wp:paragraph -->

<!-- wp:paragraph {"fontSize":"small"} -->
<p class="has-small-font-size"><strong>— The Newspaper</strong></p>
<!-- /wp:paragraph --></div>
<!-- /wp:column --></div>
<!-- /wp:columns -->

<!-- wp:spacer -->
<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->}
```

**Film Poster**

```
{<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"30px","right":"30px","bottom":"30px","left":"30px"}}},"backgroundColor":"background"} -->
<div class="wp-block-group alignwide has-background-background-color has-background" style="padding-top:30px;padding-right:30px;padding-bottom:30px;padding-left:30px"><!-- wp:cover {"url":"https://videomakerdemo.files.wordpress.com/2021/10/working_at_dawn.jpg","id":61,"dimRatio":0,"overlayColor":"background","minHeight":1410,"minHeightUnit":"px","contentPosition":"center center","align":"wide","style":{"spacing":{"padding":{"top":"20px","right":"20px","bottom":"20px","left":"20px"}}}} -->
<div class="wp-block-cover alignwide has-background-background-color" style="padding-top:20px;padding-right:20px;padding-bottom:20px;padding-left:20px;min-height:1410px"><img class="wp-block-cover__image-background wp-image-61" alt="" src="https://videomakerdemo.files.wordpress.com/2021/10/working_at_dawn.jpg" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:image {"align":"center","id":51,"width":206,"height":105,"sizeSlug":"full","linkDestination":"none"} -->
<div class="wp-block-image"><figure class="aligncenter size-full is-resized"><img src="https://videomakerdemo.files.wordpress.com/2021/10/new_film.png" alt="" class="wp-image-51" width="206" height="105"/></figure></div>
<!-- /wp:image -->

<!-- wp:paragraph {"align":"center","fontSize":"normal"} -->
<p class="has-text-align-center has-normal-font-size"><em>“Absurdly and assuredly packed punch in the gut.”</em></p>
<!-- /wp:paragraph -->

<!-- wp:paragraph {"align":"center","style":{"typography":{"fontSize":"12px"}}} -->
<p class="has-text-align-center" style="font-size:12px"><strong>Latest Review</strong></p>
<!-- /wp:paragraph -->

<!-- wp:spacer {"height":500} -->
<div style="height:500px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:paragraph {"align":"center","placeholder":"Write title…","style":{"typography":{"lineHeight":"1.3"}},"fontSize":"huge"} -->
<p class="has-text-align-center has-huge-font-size" style="line-height:1.3"><strong>MAN WORKING</strong><br><strong>AT DAWN</strong></p>
<!-- /wp:paragraph -->

<!-- wp:paragraph {"align":"center","style":{"typography":{"lineHeight":"1"}}} -->
<p class="has-text-align-center" style="line-height:1">A Film by <strong>Gary Han</strong></p>
<!-- /wp:paragraph --></div></div>
<!-- /wp:cover -->

<!-- wp:columns -->
<div class="wp-block-columns"><!-- wp:column -->
<div class="wp-block-column"><!-- wp:paragraph {"align":"center","fontSize":"medium"} -->
<p class="has-text-align-center has-medium-font-size">AKACHUKWU ELBA</p>
<!-- /wp:paragraph --></div>
<!-- /wp:column -->

<!-- wp:column -->
<div class="wp-block-column"><!-- wp:paragraph {"align":"center","fontSize":"medium"} -->
<p class="has-text-align-center has-medium-font-size">VICTORIA EDEM</p>
<!-- /wp:paragraph --></div>
<!-- /wp:column -->

<!-- wp:column -->
<div class="wp-block-column"><!-- wp:paragraph {"align":"center","fontSize":"medium"} -->
<p class="has-text-align-center has-medium-font-size">MUSA IDOWU</p>
<!-- /wp:paragraph --></div>
<!-- /wp:column -->

<!-- wp:column -->
<div class="wp-block-column"><!-- wp:paragraph {"align":"center","fontSize":"medium"} -->
<p class="has-text-align-center has-medium-font-size">SANI GODWIN</p>
<!-- /wp:paragraph --></div>
<!-- /wp:column --></div>
<!-- /wp:columns -->

<!-- wp:columns {"align":"wide"} -->
<div class="wp-block-columns alignwide"><!-- wp:column {"width":"100%"} -->
<div class="wp-block-column" style="flex-basis:100%"><!-- wp:paragraph {"align":"center","style":{"typography":{"fontSize":"12px"}}} -->
<p class="has-text-align-center" style="font-size:12px">Videomaker Collective Presents In Association with HANO Direction by Dan Andrews Screenplay by Augusta Smith Camera Diego Cruz, Helen Smith-Andrews and Dan Andrews Art Philip Mizrahi Production Romana Arakelyan Operators Syd Wyatt and Cameron Madeira Behind the Camera Diego Cruz Color Editing and Post-Production Philip Mizrahi and Tony Sánchez Videomaker Collective Presents In Association with HANO Direction by Dan Andrews Screenplay by Augusta Smith Camera Diego Cruz, Helen Smith-Andrews and Dan Andrews Art Philip Mizrahi Production Romana Arakelyan Operators Syd Wyatt and Cameron Madeira Behind the Camera Diego Cruz Color Editing and Post-Production Philip Mizrahi and Tony Sánchez</p>
<!-- /wp:paragraph --></div>
<!-- /wp:column --></div>
<!-- /wp:columns --></div>
<!-- /wp:group -->}
```

**About**

```
{<!-- wp:spacer -->
<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->

<!-- wp:columns {"align":"wide"} -->
<div class="wp-block-columns alignwide"><!-- wp:column -->
<div class="wp-block-column"><!-- wp:heading {"level":3,"style":{"typography":{"fontWeight":"300"}},"fontSize":"large"} -->
<h3 class="has-large-font-size" style="font-weight:300">Gary Han is a award-winning creative director, designer, photographer and visual artist based in Singapore.</h3>
<!-- /wp:heading --></div>
<!-- /wp:column --></div>
<!-- /wp:columns -->

<!-- wp:columns {"align":"wide"} -->
<div class="wp-block-columns alignwide"><!-- wp:column -->
<div class="wp-block-column"><!-- wp:image {"id":93,"sizeSlug":"large","linkDestination":"none"} -->
<figure class="wp-block-image size-large"><img src="https://videomakerdemo.files.wordpress.com/2021/10/team-1-edited.jpg" alt="" class="wp-image-93"/></figure>
<!-- /wp:image -->

<!-- wp:paragraph {"fontSize":"medium"} -->
<p class="has-medium-font-size"><a href="https://videomakerdemo.wordpress.com/about/" data-type="page" data-id="13">The Crew ↗</a></p>
<!-- /wp:paragraph --></div>
<!-- /wp:column -->

<!-- wp:column -->
<div class="wp-block-column"><!-- wp:image {"id":94,"sizeSlug":"large","linkDestination":"none"} -->
<figure class="wp-block-image size-large"><img src="https://videomakerdemo.files.wordpress.com/2021/10/philosophy-1-edited.jpg" alt="" class="wp-image-94"/></figure>
<!-- /wp:image -->

<!-- wp:paragraph {"fontSize":"medium"} -->
<p class="has-medium-font-size"><a href="https://videomakerdemo.wordpress.com/about/" data-type="page" data-id="13">Our Philosophy ↗</a></p>
<!-- /wp:paragraph --></div>
<!-- /wp:column -->

<!-- wp:column -->
<div class="wp-block-column"><!-- wp:image {"id":88,"sizeSlug":"large","linkDestination":"none"} -->
<figure class="wp-block-image size-large"><img src="https://videomakerdemo.files.wordpress.com/2021/10/blog-edited.jpg" alt="" class="wp-image-88"/></figure>
<!-- /wp:image -->

<!-- wp:paragraph {"fontSize":"medium"} -->
<p class="has-medium-font-size"><a href="https://videomakerdemo.wordpress.com/blog/" data-type="page" data-id="3">Blog ↗</a></p>
<!-- /wp:paragraph --></div>
<!-- /wp:column --></div>
<!-- /wp:columns -->

<!-- wp:spacer -->
<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->}
```

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
This adds the patterns from https://github.com/Automattic/themes/issues/4859:
localhost_4759__p=869

#### Related issue(s):
Fixes https://github.com/Automattic/themes/issues/4859

### CODE REVIEW

**Comment by MaggieCabrera** [comment]
mmh, is this missing files? I only see the pattern called "description".

**Comment by scruffian** [comment]
Sorry, I forgot to add them. They should be there now. 

I had to recreate them so I might have missed something this time :)

**Comment by MaggieCabrera** [comment]
The `About` images seem to be really big in resolution

File: videomaker/inc/patterns/description.php [hunk 1]
```diff
@@ -8,8 +8,29 @@
 return array(
 	'title'      => __( 'Description', 'videomaker' ),
 	'categories' => array( 'videomaker' ),
-	'content'    => '<!-- wp:heading {"align":"wide","style":{"typography":{"fontWeight":"300"}}} -->
-	<h2 class="alignwide" style="font-weight:300">' . esc_html__( 'Gary Han is a award-winning creative director, designer, photographer and visual artist based in Singapore.', 'videomaker' ) . '</h2>
-	<!-- /wp:heading -->',
+	'content'    => '<!-- wp:spacer {"height":50} -->
+	<div style="height:50px" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer -->
```
Comment (MaggieCabrera) [inline videomaker/inc/patterns/description.php:13]: Are we replacing the old "Description" pattern here?

**Review by MaggieCabrera — APPROVED** [review]
LGTM pending the two comments that I made, the patterns look good and so does the code.

**Comment by beafialho** [comment]
A few comments:

- Can we remove the word "videomaker" after the text in the `About` pattern?
Captura de ecrã 2021-10-29, às 16 19 59

- I mistakenly set the line heights of the paragraphs _"Based on the novel “Man Working at Dawn” by"_, _"Screenplay by"_ and _"Directed by"_ to 0,5. Let's make that 1,3, so it doesn't overlap in smaller screens.

Captura de ecrã 2021-10-29, às 16 23 08

- Lastly, in the `Film Poster` pattern, let's keep the "Man Working at Dawn" heading in the same line and set it to 45, so it'll look good on mobile.
Captura de ecrã 2021-10-29, às 16 24 19

Should I paste the resized images from the `About` pattern here?

File: videomaker/inc/patterns/description.php (hunk 1 above)
Comment (scruffian) [reply videomaker/inc/patterns/description.php:13]: I think so yeah, the old one was almost empty

**Comment by scruffian** [comment]
I updated the patterns and images. I'm going to merge this. If there are more issues, we can fix them in a followup PR.

**Pull request merged**